### DISCUSSION TITLE
How to use httpsnippet - Codegen

### DISCUSSION THREAD

**Discussion by st11x** [thunderclient/thunder-client-support#796] [Q&A]
Hi,

On this page

https://github.com/rangav/thunder-client-support#codegen

it says other languages are powered by httpsnippet. How do you actually integrate it? 

I do not have node installed and only use node docker images for any node related projects.

thanks
   Matt

**Comment by rangav** [comment]
@st11x go to their GitHub which has instructions and if you have further questions about that library, better to ask them.
